don't check for libbsd and libelf, they aren't needed anywhere.

--- mico/configure.in
+++ mico/configure.in
@@ -1237,9 +1237,7 @@
 
 
 AC_CHECK_LIB(socket, open)
-AC_CHECK_LIB(bsd, open)
 
-AC_CHECK_LIB(elf, open)
 AC_CHECK_LIB(dl, open)
 AC_CHECK_LIB(dld, open)
 AC_CHECK_LIB(ld, open)
